src/mlj_catboostclassifier.jl

```diff
@@ -0,0 +1,134 @@
+
```

Comment:
What are all these `nothing`s ? Why aren't proper default values being set here?

Reply:
I wanted it to default to the python library's default parameters. My thought process was that it was lower maintenance, and it avoids getting different default parameters if the python version changes things up. 

However, I'll gladly add them in if you think they should be included.

Reply:
Yeah, I realize this is a maintenance headache. I've had the same complaint about XGBoost. But this seems particular to the wrapped gradient tree boosters - other models don't have many parameters and defaults rarely change. MLJ kind of needs this, at least at the moment, for multiple reasons. For example, it's good to know what the default type of a field value is when constructing hyperparameter ranges for tuning. 

So, can you add them please? 🙏🏾

Reply:
I have added them. There are quite a few dynamic parameters in CatBoost that don't have universal default values. I have kept them as `nothing` for now.

Reply:
Yeah that should be okay.